test/e2e_node/hugepages_test.go

```diff
@@ -230,7 +230,7 @@ var _ = SIGDescribe("HugePages", framework.WithSerial(), feature.HugePages, "[No
 			framework.ExpectNoError(err, "while getting node status")
 			_, isPresent := node.Status.Capacity["hugepages-3Mi"]
 			return isPresent
-		}, 30*time.Second, framework.Poll).Should(gomega.BeFalse())
+		}, 30*time.Second, framework.Poll).Should(gomega.BeFalseBecause("expected the hugepages-2Mi resource to be absent"))
```

Comment:
Instead of returning `isPresent`, return `node.Status.Capacity` and check for it with a gomega matcher.